pkgs/development/tools/udis86/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, python }:
+
+stdenv.mkDerivation rec {
+  pname = "udis86";
+  version = "1.7.2";
+
+  src = fetchFromGitHub {
+    owner = "vmt";
+    repo = "udis86";
+    rev = "v${version}";
+    url = "https://github.com/vmt/udis86/archive/v${version}.tar.gz";
+    sha256 = "0c60zwimim6jrm4saw36s38w5sg5v8n9mr58pkqmjrlf7q9g6am1";
+  };
+
+  nativeBuildInputs = [ autoreconfHook python ];
+
+  configureFlags = [
+    "--enable-shared"
+  ];
+
+  outputs = [ "bin" "out" "dev" "lib" ];
+
+  meta = with stdenv.lib; {
```

Comment:
@timor I'm sorry I'm not sure how I missed this but you need to include `platforms` please.

Reply:
I have no other system than `x86_64-linux` available to test this. What is the policy: Should I just assume it builds on all platforms, or should I only specify the ones I tested on?

Reply:
@timor unless you have reason to believe it won't build on our 3 major platforms you can just set to `platforms.all` and the bot will let you know if that isn't the case.

Reply:
Thanks for the clarification.  I added the platform information.